app.env.example

```diff
@@ -2,27 +2,47 @@
 DB_HOST=db
 DB_USER=postgres
 DB_PWD=
+BASE_URL=http://localhost:3000
 ALLOWED_HOSTS= --- comma separated ip ---
+FB_NOTIFY_ENABLED=false
+TG_NOTIFY_ENABLED=false
+FB_REACHABLE_DAY=1
+FB_REACHABLE_IN_DAY=7
 RAILS_MAX_THREADS=5
+CC_TEST_REPORTER_ID=
+VERBOICE_USER=
+VERBOICE_PWD=
 REDIS_URL=redis://redis:6379
-BASE_URL=http://localhost:3000
-CC_TEST_REPORTER_ID=7ae1359b52f2b8ff66704799670aadd3be2a22c9699fde4a675e432bb34fe9e8
-
 GUISSO_CLIENT_ID= --- REGISTER APP AT login.instedd.org ---
 GUISSO_CLIENT_SECRET= --- REGISTER APP AT login.instedd.org ---
 SENTRY_DSN=
-
-VERBOICE_USER=verboice_user
-VERBOICE_PWD=verboice_pwd
 FB_MESSAGING_URL=https://graph.facebook.com/v6.0/me/messages
 FB_MESSAGING_TOKEN=FB_PAGE_TOKEN
 TG_MESSAGING_URL=https://api.telegram.org
 TG_MESSAGING_TOKEN=TELEGRAM_TOKEN
-FB_NOTIFY_ENABLED=false
-FB_REACHABLE_DAY=1
 TELEGRAM_ENABLED=false
+APP_VERSION=1.0
 MAX_CSV_DOWNLOAD_SIZE=10000
+DASHBOARD_START_DATE=2020/08/28
+HOMEPAGE_START_DATE=2020/08/28
 
-APP_VERSION=1.0
 # Shell with unicode support
 LANG=C.UTF-8
+
+#aws
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=
+AWS_S3_URL=
+
+#static website
+ENDPOINT_URL=https://ow4c-dashboard-stg.ilabsea.org/
+TOKEN=abc123
+RECAPTCHA_SITE_KEY=
+RECAPTCHA_SECRET_KEY=
+
+FACEBOOK_PAGE_ID=
+FACEBOOK_APP_ID=
+
+PILOT_PROVINCES=02,04
```

Comment:
@radinreth use PILOT_PROVINCE_CODES instead and give comment # with a link to the province code database on NCDD's web site